_posts/en/pages/faq/2016-01-21-rbf-faq.md

```diff
@@ -0,0 +1,237 @@
+
+##I heard Opt-in RBF was added with little or no discussion
+
+Recent RBF discussions going back to May 2015.
+
+Github:
+- Add first-seen-safe replace-by-fee logic to the mempool [#6176](https://github.com/bitcoin/bitcoin/pull/6176)
+- Scheduled full-RBF deployment [#6352](https://github.com/bitcoin/bitcoin/pull/6352)
+- nSequence-based Full-RBF opt-in [#6871](https://github.com/bitcoin/bitcoin/pull/6871)
+
+IRC meeting:
+https://www.reddit.com/r/Bitcoin/comments/3t1in5/bitcoin_dev_irc_meeting_in_laymans_terms_20151112/
+
+There are many other logs which can be found at http://bitcoinstats.com/ for #bitcoin-dev and https://botbot.me/freenode/bitcoin-core-dev/ for #bitcoin-core-dev
+
+Mailing list discussions in no particular order:
+
+- [[bitcoin-dev] Opt-in Full Replace-By-Fee (Full-RBF)](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-November/011783.html)
+- [[Bitcoin-development] First-Seen-Safe Replace-by-Fee](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-May/008248.html)
+- [[Bitcoin-development] Cost savings by using replace-by-fee, 30-90%](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-May/008232.html)
+- [[bitcoin-dev] Significant losses by double-spending unconfirmed transactions](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-July/009420.html)
+- [[bitcoin-dev] BIP: Full Replace-by-Fee deployment schedule](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-June/009253.html)
+- [[Bitcoin-development] replace-by-fee v0.10.0rc4](http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-February/007404.html)
+- [[bitcoin-dev] How wallets can handle real transaction fees](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-November/011685.html)
+
+Satoshi originally introduced [unconfirmed transaction replacement](https://github.com/trottier/original-bitcoin/blob/master/src/main.cpp#L434)
+but this was disabled due to a DOS attack (which Peter Todd fixed by requiring a higher fee for each replacement).
+
+## Can this be used to double spend against unprepared (old) wallets? Will all wallets have to update?
+
+> Can this be used to double spend against unprepared (old) wallets?
+
+Unconfirmed transactions can always be double spent, with or without RBF. RBF doesn't make fraud any easier (it's already easy), only legitimate double spending. So the answer is "yes", but it was already "yes" before RBF, and there is nothing wallets can do to change it to a "no" either way.
+
+> Will all wallets have to update?
+
+Only to make use of it. This opens up a lot of room to improve wallets.
+
+## Why not FSS-RBF? (First-seen-safe Replace-by-fee)
+
+First-seen-safe means that transactions can only be modified if all previous outputs are fully spent. This prevents double spends.
+
+The problem with FSS is when you spend money you are signing the "change" (think like quarters, nickles, type change) over back to yourself.
```

Comment:
It might be highlighted that, in case of full blocks, FSS-RBF only adds fuel to the fire and makes congestion worse, due to the inherently larger transactions as result.